xchange-binance/src/main/java/org/knowm/xchange/binance/dto/account/WithdrawRequest.java

```diff
@@ -2,19 +2,14 @@
 
 import com.fasterxml.jackson.annotation.JsonProperty;
 
-public final class WithdrawRequest extends ApiResponse<String> {
-  public final String id;
+public final class WithdrawRequest {
```

Comment:
I think it would be better to rename this to `WithdrawResponse`

Reply:
renamed